app/services/GameMenuPresenter.java

```diff
@@ -0,0 +1,34 @@
+package services;
+
+import ttt.game.GameOption;
+
+import java.util.HashMap;
+
+public class GameMenuPresenter implements MenuPresenter {
+
+    private String endpoint;
+
+    public GameMenuPresenter() {
+        this.endpoint = "/new-game";
+    }
+
+    public HashMap<String, String> showOptions() {
+        HashMap<String, String> options = new HashMap<>();
+        for (GameOption gameOption : GameOption.values()) {
+            options.put(gameOption.key, gameOption.title);
+        }
+        return options;
+    }
+
+    public String getEndpoint() {
+        return endpoint;
+    }
+
+    public boolean isChosen() {
+        return false;
```

Comment:
Why does this always return false? Is it to say that it is the computer turn not the human's? If so, maybe a more descriptive name would be helpful.  Or is it because these methods are not really required, but the interface dictates that they are needed? If the latter - perhaps this is a case of ISP?